### DISCUSSION TITLE
flat_priors option development

### DISCUSSION THREAD

**Discussion by smwindecker** [idem-lab/lowerGPreff#34] [General]
one option is as implemented in current flat_priors branch (poisson observation model so only one param needs to be estimated). 

one way to diagnose if the 2 parameters in the negative binomial observation model are a problem is to get posteriors of the size parameter, and fix those values for the size during fitting, and check convergence. 

notes from Nick: re Poisson obs model solution, it could well be that the extra degrees of freedom in the process model mean they aren't needed in the observation model. The rigorous way to resolve this would be: are the residual assumptions in a posterior predictive simulation analysis? I would expect bad convergence because of the very large parameter space and difficulty initialising all the daily infection incidence parameters in a reasonable part of parameter space, so finding some good initial values for them (e.g. from posterior mean of smoothed model) would be a good solution. Different issue to the likelihood part.

**Comment by AugustHao** [comment]
just reading Nick's comment, I think it makes sense to try with an informative initial value. An alternative approach to using posterior mean of smoothed model is to take the case numbers, shift them back in time by mean delay days, which should roughly give you a good starting value for infection numbers (assuming perfect ascertainment). The advantage of this is you don't need to run the smoothed model beforehand to get initial values. We should then add a very small random error to the initial values for each chain, so that they don't start from exactly the same point.

**Comment by AugustHao** [comment]
re extra degrees of freedom in process model making overdispersion redundant in the observation model - I think that is what Cam wants? ie more variance attributed to the infection process rather than observation process. Best to check with them if this approach (ie Poisson distribution over case counts) is consistent with their motivation for temporally independent flat prior infection model

**Comment by AugustHao** [comment]
another point: if we are fitting Poisson distribution over case counts (case_t), with temporally independent flat priors for infection on each day (let's called it lambda_t) , and a **fixed constant delay** _d_ from infection to case counts. Then, for each day t, the posterior distribution for lambda_t is simply Poisson(case_(t+d)), because there is only one observation case_(t+d) to inform this parameter. This makes me wonder if it is simply easier to algebraically deconvolve the infection count timeseries (or approximate it with a deterministic solution) to get the temporally independent infection count distribution, rather than going through the MCMC process.
